opencti-platform/opencti-graphql/src/modules/workspace/workspace-domain.ts

```diff
@@ -77,6 +82,30 @@ export const objects = async (context: AuthContext, user: AuthUser, workspaceId:
   return listThings(context, user, types, finalArgs);
 };
 
+export const exportObjectsAsStixReportBundle = async (context: AuthContext, user: AuthUser, workspace: BasicStoreEntityWorkspace): Promise<string> => {
+  if (workspace.type !== 'investigation') {
+    throw FunctionalError('You can only export investigation objects as a stix report bundle');
+  }
+  const types = ['Stix-Core-Object', 'Stix-Core-Relationship', 'Stix-Sighting-Relationship'];
+  const filters = [{ key: buildRefRelationKey(RELATION_HAS_REFERENCE), values: [workspace.id] }];
+  const stixObjects = await stixLoadByFilters(context, user, types, { filters });
+  const reportData = {
+    name: workspace.name,
+    published: workspace.created_at,
+    type: 'report',
+    object_refs: stixObjects.map((s) => s.id),
+    spec_version: STIX_SPEC_VERSION,
+  };
+  const report = { ...reportData, id: generateStandardId(ENTITY_TYPE_CONTAINER_REPORT, reportData) };
+  const allObjects = [...stixObjects, report];
+  const bundle = {
+    id: `bundle--${uuidv4()}`,
+    type: 'bundle',
+    objects: allObjects,
+  };
+  return JSON.stringify(bundle);
```

Comment:
@richard-julien Do we want to export a json rather than a string so it can be used directly via the API?

Reply:
@Kedae maybe an opinion on that ?